setup.py

```diff
@@ -266,6 +267,9 @@ def make_distribution(self):
 script_files = glob.glob("scripts/*")
 install_requires = ["numpy"]
 setup_requires = ["numpy"]
+extras_require = {"tango": ["pytango"]}
```

Comment:
With this, pytango does not become a strong dependency of the project ?

Reply:
indeed, you need to do `pip install .[tango]` to install the tango dependencies (which is just pytango).